Makefile

```diff
@@ -46,4 +46,19 @@ check_trufflehog:
 trufflehog: check_trufflehog
 	trufflehog --regex --rules .trufflehog.json --entropy=False file:///$$azul_home
 
-.PHONY: all hello terraform deploy subscribe everything reindex clean test travis integration_test trufflehog check_trufflehog delete
+check_clean:
+	git diff --exit-code  && git diff --cached --exit-code
+
+check_autosquash:
+	_azul_target_branch="$${TRAVIS_BRANCH:=develop}" \
+	; _azul_merge_base=$$(git merge-base HEAD "$${_azul_target_branch}") \
+	; if GIT_SEQUENCE_EDITOR=: git rebase -i --autosquash "$${_azul_merge_base}"; then \
+	    git reset --hard ORIG_HEAD \
+	    ; true \
```

Comment:
Consider echoing a string in either case explaining the conclusion.

Reply:
I'll add one for the negative case. The positive case should not produce output, following the Unix Mantra of not producing extraneous output.

Reply:
I'll add one for the negative case. The positive case should not produce output, following the Unix Mantra of not producing extraneous output.

Reply:
I added the message for the positive case and silenced the echoing of the make rule.